docs/content/basics/testing_theory.md

```diff
@@ -1,59 +1,171 @@
 # Testing theory
 
 
-This article will cover the most primitive things that will allow you to understand better the terms and general concepts regarding testing.
+This article will cover the most primitive things that will allow you to understand better the terms and general concepts regarding automated testing.
```

Comment:
if we speak about that topic it's not only about `automated`, for example we do have unit testing explanation in that  article

Reply:
I removed the manual testing part, but I will add it again, makes sense to me